### PR TITLE
MWPW-118022 - Improve Modal Accessibility

### PR DESCRIPTION
**Pull request by vhargrave**
* Addressed lack of focus indicator on close button
* Moved close button higher in the append order to get free focus

Resolves: [MWPW-118022](https://jira.corp.adobe.com/browse/MWPW-118022)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/docs/authoring/how-to-videos#create-page?martech=off
- After: https://focus-modals-vhargrave--milo--adobecom.hlx.page/docs/authoring/how-to-videos#create-page?martech=off

### CODE REVIEW

File: libs/blocks/modal/modal.css [hunk 1]
```diff
@@ -1,3 +1,8 @@
+:root {
+  /* Modal Colors */
```
Comment (chrischrischris) [inline libs/blocks/modal/modal.css:2]: nit: no need for the comment

**Review by chrischrischris — COMMENTED** [review]
First PR!  Congrats! 

A few minor things:
* In the PR description, delete the generic text if you aren't going to replace it
* For the Before/After urls, please point to a page that can show what your changes are.  In this case it would be a page that opens a modal.  This allows for users to test and verify your changes.
* PR title should start with jira number, and a descriptive title

Otherwise looks great.

File: libs/blocks/modal/modal.css (hunk 1 above)
Comment (vhargrave) [reply libs/blocks/modal/modal.css:2]: Makes sense. I'll get of that . Thanks!

Comment (vhargrave) [reply libs/blocks/modal/modal.css]: Thanks also for the feedback on the PR description. I'm having a few difficulties getting the links to work, so I'll have to check that out when tomorrow.

**Review by auniverseaway — APPROVED** [review]
Seeing some odd behavior in Chrome, but it's also reproducible in Dexter. I think this is fine.

**Review by rgclayton — APPROVED** [review]

**Comment by auniverseaway** [comment]
@Brandon32 @meganthecoder this is a good one for your info: the test page is firing a modal so performance is going to be terrible. It's not indicative of a real-world scenario we would run LH against.

**Pull request merged**